planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -50,6 +50,22 @@ using tier4_autoware_utils::calcDistance2d;
 using tier4_autoware_utils::calcLateralDeviation;
 using tier4_planning_msgs::msg::AvoidanceDebugFactor;
 
+namespace
+{
+
+AvoidLine getNotStraightShiftLine(const AvoidLineArray & shift_lines)
```

Comment:
Non-straight sounds more natural
```suggestion
AvoidLine getNonStraightShiftLine(const AvoidLineArray & shift_lines)
```

Reply:
fix in https://github.com/autowarefoundation/autoware.universe/pull/2533/commits/980437c70ae3c43ed25684fbe868e019c5323d7c

Reply:
Thanks!